training/dist_clm_train.py

```diff
@@ -163,11 +166,6 @@ def train_loop(args, pipe, device, train_data_loader, test_data_loader):
                 if dp_rank == 0:
                     save_checkpoint(pipe, args)
 
-                    if event_reporter is not None:
-                        event_reporter.report(object=EventReporter.OBJECT_FINE_TUNE,
-                                              message=f"checkpoint saved for step {pipe.global_step}",
-                                              event_type=EventReporter.EVENT_TYPE_CHECKPOINT_SAVE,
-                                              requires_is_enabled=False)
```

Comment:
Why are we removing this?

Reply:
This is going to be handled by ft job script.